pkg/preparation/storacha/storacha.go

```diff
@@ -77,10 +95,27 @@ func (a API) SpaceBlobAddShardsForUpload(ctx context.Context, uploadID id.Upload
 			locationCommitment,
 		)
 		if err != nil {
-			return fmt.Errorf("failed to replicate shard %s: %w", shard.CID(), err)
+			return fmt.Errorf("failed to replicate shard %s: %w", shard.ID(), err)
+		}
+
+		pieceDigest, _, err := commpCalc.Digest()
```

Comment:
Do we surface this anywhere?

`filecoin/info` uses the piece CID to look up the current deals and inclusion proof.

Reply:
Surface as in show the Piece CID to the user? Not currently, in Guppy. It appears in Console, so I assumed we'd eventually make a way to look it up in Guppy as well. Might be worth printing it, though. So far we aren't printing any information about individual shards, except as part of the progress screen.